.github/workflows/acceptance-tests.yml

```diff
@@ -1,4 +1,5 @@
 name: 'Acceptance Tests'
+run-name: 'Acceptance Tests ${{ inputs.atlas_cloud_env }} ${{ inputs.test_group }}'
```

Comment:
We should update the RELEASING.md if we plan to manually run this as part of the release process. (feel free to ignore if no decision has been made yet)